recipe/run_test.py

```diff
@@ -0,0 +1,23 @@
+import subprocess
+
+import fastapi
+from fastapi import FastAPI, dependencies, middleware, openapi, security
+from fastapi.testclient import TestClient
+
+
+subprocess.run(["pip", "check"])
+
+app = FastAPI()
+
+
+@app.get("/")
+async def read_main():
+    return {"msg": "Hello World"}
+
+
+client = TestClient(app)
+
+def test_read_main():
```

Comment:
I think we should run the upstream unit tests instead of writing a test ourselves. It'll be easier to maintain.